src/commands/createCosmosDBAccount.ts

```diff
@@ -70,38 +70,28 @@ async function createResourceGroup(subscriptionNode: IAzureNode): Promise<Resour
                 const resourceManagementClient = new ResourceManagementClient(subscriptionNode.credentials, subscriptionNode.subscription.subscriptionId);
```

Comment:
I assume you've tested all the changed scenarios?  I wonder if it's not better to keep the current behavior, though, instead of throwing exceptions.

Reply:
This will probably change anyways once we move the ui stuff to the shared package (https://github.com/Microsoft/vscode-azuretools/issues/23) something I'd like to do soon after this release